source/rust_verify/tests/functions.rs

```diff
@@ -0,0 +1,30 @@
+#![feature(rustc_private)]
+#[macro_use]
+mod common;
+use common::*;
+
+test_verify_one_file! {
+    #[test] test_use_fun_ext verus_code! {
+        use crate::pervasive::function::*;
+
+        proof fn test_use_fun_ext(f: FnSpec(int) -> int) {
+          fun_ext::<int, int>(|i: int| i + 1, |i: int| 1 + i);
```

Comment:
All this tests is the requires line, right? Is there any useful thing to test about the ensures?

Reply:
I could add `assert ((|i: int| i + 1) === (|i: int| 1 + i));` to this test to use the postcondition, although the postcondition is also tested by the other test in this PR currently.